builder/chroot/builder.go

```diff
@@ -140,7 +140,8 @@ type Config struct {
 	//This will fail unless *exactly* one AMI is returned. In the above example,
 	//`most_recent` will cause this to succeed by selecting the newest image.
 	//
-	//-   `filters` (map of strings) - filters used to select a `source_ami`.
+	//-   `filters` (map{key string, value string | comma separated string}) -
```

Comment:
Only suggestion here would be to make this `map[string]string` and as part of the comment specify that the value can be a comma separated list with an example.

Reply:
I will look at this today, sorry I haven't addressed sooner